README.md

```diff
@@ -285,11 +304,11 @@ practice which also met our design objective to keep the certificate request pro
 VCert users. If you require the ability to specify the CA Template with the request you can use the
 TPP REST APIs but please be advised this goes against Venafi recommendations.
 
-## Prerequisites for using with Venafi Cloud
+## Prerequisites for using with Venafi as a Service
 
-1. The Venafi Cloud REST API is accessible at https://api.venafi.cloud from the system where VCert
+1. The Venafi as a Service REST API is accessible at https://api.venafi.cloud or https://api.eu.venafi.cloud(if you have an EU account) from the system where VCert
```

Comment:
missing a blank space between the url and the parenthesis
```
https://api.eu.venafi.cloud (if you have an EU account)
```

Reply:
Done!